### PR TITLE
DX-1188 Add code comments to describe env vars in .env file

### PR DESCRIPTION
**Pull request by marcospinello**
* Add code comments to describe env vars in Vite `.env` file: a follow-up to issue https://github.com/miroapp/app-examples/issues/80
* Add link to the complementary complete tutorial on dev docs
* Hyphenate "full stack" to "full-stack" when it is used as an adjective/modifier

### CODE REVIEW

File: examples/github-appcards/README.md [hunk 1]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
+
+# Enter here the API key of the Supabase project with the database that the app uses for data persistence.
+# For more information, see:
+# https://supabase.com/docs/guides/api#api-url-and-keys
 VITE_DATABASE_PUBLIC_KEY=
+
+# Enter here the base URL of the hosting service your app runs on.
+# If you're developing locally, it can be 'localhost'.
 VITE_BASE_URL=
-VITE_MIRO_API_TOKEN=
```
Comment (janza) [inline examples/github-appcards/README.md:63]: Was this one removed on purpose?

**Review by janza — APPROVED** [review]

File: examples/github-appcards/README.md [hunk 2]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
 
 ## Required Environment Variables
 
+To run the app, Vite loads an `.env` file with the [environment variables](https://vitejs.dev/guide/env-and-mode.html#env-files).
+Use the commented example below to create your `.env` file.
+
 ```.env
+# Generate an access token in GitHub, and enter the value here.
+# To generate the access token, go to https://github.com/settings/tokens
```
Comment (mettin) [inline examples/github-appcards/README.md:66]:
```suggestion
# [Generate an access token in GitHub](https://github.com/settings/tokens), and enter the value here.
```

File: examples/github-appcards/README.md [hunk 3]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
 VITE_GH_ACCESS_TOKEN=
+
+# In Supabase, you create a Postgres database, and you assign it a password.
+# Enter here the password of the database created in Supabase.
```
Comment (mettin) [inline examples/github-appcards/README.md:70]:
```suggestion
# Enter the password of the created database in Supabase here.
```

File: examples/github-appcards/README.md [hunk 4]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
+# For more information, see:
+# https://supabase.com/docs/guides/database/managing-passwords
 VITE_SUPABASE_PASSWORD=
+
+# Enter here the URL of the Supabase database that the app uses for data persistence.
```
Comment (mettin) [inline examples/github-appcards/README.md:75]:
```suggestion
# Enter the URL of the Supabase database that the app uses for data persistence here.
```

File: examples/github-appcards/README.md [hunk 5]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
+# For more information, see:
+# https://supabase.com/docs/guides/database
 VITE_DATABASE_URL=
+
+# Enter here the API key of the Supabase project with the database that the app uses for data persistence.
```
Comment (mettin) [inline examples/github-appcards/README.md:80]:
```suggestion
# Enter the API key of the Supabase project with the database that the app uses for data persistence here.
```

File: examples/github-appcards/README.md [hunk 6]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
+# For more information, see:
+# https://supabase.com/docs/guides/api#api-url-and-keys
 VITE_DATABASE_PUBLIC_KEY=
+
+# Enter here the base URL of the hosting service your app runs on.
```
Comment (mettin) [inline examples/github-appcards/README.md:85]:
```suggestion
# Enter the base URL of the hosting service your app runs on here.
```

File: examples/github-appcards/README.md [hunk 7]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
+# If you're developing locally, it can be 'localhost'.
 VITE_BASE_URL=
-VITE_MIRO_API_TOKEN=
+
+# Enter here the client secret of your app.
```
Comment (mettin) [inline examples/github-appcards/README.md:89]:
```suggestion
# Enter the client secret of your app here.
```

File: examples/github-appcards/README.md [hunk 8]
```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
...
+# For more information, see:
+# https://developers.miro.com/docs/build-your-first-hello-world-app#step-3-configure-your-app-in-miro
 MIRO_CLIENT_SECRET=
+
+# Enter here the OAuth code grant flow redirect URI for your app.
```
Comment (mettin) [inline examples/github-appcards/README.md:94]:
```suggestion
# Enter the OAuth code grant flow redirect URI for your app here.
```

**Review by mettin — APPROVED** [review]

File: examples/github-appcards/README.md (hunk 1 above)
Comment (marcospinello) [reply examples/github-appcards/README.md:63]: @janza yes, based on this comment in https://github.com/miroapp/app-examples/issues/80:

```
# Ignore this field, it's no longer necessary. We'll update the README accordingly.
VITE_MIRO_API_TOKEN=
```

We don't use this env var anywhere inside this repo.
Do you know where else I can check to make sure we really don't need it?

File: examples/github-appcards/README.md (hunk 2 above)
Comment (marcospinello) [reply examples/github-appcards/README.md:66]: Markdown markup does not work inside code blocks, it renders as literal

**Review by mettin — APPROVED** [review]

File: examples/github-appcards/README.md (hunk 1 above)
Comment (marcospinello) [reply examples/github-appcards/README.md:63]: Update:
`VITE_MIRO_API_TOKEN` is not used anywhere. It's safe to remove it from the `.env` file.

**Pull request merged**